### PR TITLE
Update Job executions

### PR DESCRIPTION
**Pull request by FlorianJacta**

### CODE REVIEW

File: docs/knowledge_base/tutorials/scenario_management_overview/step_07/index.md
Comment (jrobinAV) [inline docs/knowledge_base/tutorials/scenario_management_overview/step_07/index.md:1]: is it still relevant to download all steps?

File: docs/knowledge_base/tutorials/scenario_management_overview/step_07/index.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ def add(nb):
     return nb + 10
 ```
 
-![](config_07.svg){ width=700 style="margin:auto;display:block;border: 4px solid rgb(210,210,210);border-radius:7px" }
+![](config.svg){ width=700 style="margin:auto;display:block;border: 4px solid rgb(210,210,210);border-radius:7px" }
```
Comment (jrobinAV) [inline docs/knowledge_base/tutorials/scenario_management_overview/step_07/index.md]: please provide a name to the picture. It s better for SEO
```suggestion
![Configuration](config.svg){ width=700 style="margin:auto;display:block;border: 4px solid rgb(210,210,210);border-radius:7px" }
```

**Review by jrobinAV — APPROVED** [review]

File: docs/knowledge_base/tutorials/scenario_management_overview/src/step_07.py
Comment (jrobinAV) [inline docs/knowledge_base/tutorials/scenario_management_overview/src/step_07.py:1]: Can you move the file to step_07/ folder and update the links in index.md

File: docs/knowledge_base/tutorials/scenario_management_overview/src/step_07.py [hunk 2]
```diff
@@ -33,8 +33,8 @@ def add(nb):
 
 # Configuration of the pipeline and scenario
 scenario_cfg = Config.configure_scenario(id="my_scenario",
-                                                    task_configs=[first_task_cfg,
-                                                                  second_task_cfg])
+                                         task_configs=[first_task_cfg,
+                                                       second_task_cfg])
 
 Config.export("config_07.toml")
```
Comment (jrobinAV) [inline docs/knowledge_base/tutorials/scenario_management_overview/src/step_07.py]: Can you move config_07.toml to `step_07/` folder?

File: docs/knowledge_base/tutorials/scenario_management_overview/src/step_07.py
Comment (FlorianJacta) [reply docs/knowledge_base/tutorials/scenario_management_overview/src/step_07.py:1]: Should I do all these changes in another PR?

Comment (jrobinAV) [reply docs/knowledge_base/tutorials/scenario_management_overview/src/step_07.py:1]: If the file is not referenced eslewhere than in step_07/ folder, then you can do it in this PR.

File: docs/knowledge_base/tutorials/scenario_management_overview/step_07/index.md [hunk 3]
```diff
@@ -92,20 +95,21 @@ output_cfg = Config.configure_data_node("output")
...
+                                        output_cfg)
 
 # Configuration of the pipeline and scenario
 scenario_cfg = Config.configure_scenario(id="my_scenario",
-                                                    task_configs=[first_task_cfg,
-                                                                  second_task_cfg])
+                                         task_configs=[first_task_cfg,
+                                                       second_task_cfg])
 
+Config.export("config_07.toml")
```
Comment (jrobinAV) [inline docs/knowledge_base/tutorials/scenario_management_overview/step_07/index.md:112]:
```suggestion
Config.export("config.toml")
```

File: docs/knowledge_base/tutorials/scenario_management_overview/step_07/job_execution.py [hunk 4]
```diff
@@ -33,8 +33,8 @@ def add(nb):
 
 # Configuration of the pipeline and scenario
 scenario_cfg = Config.configure_scenario(id="my_scenario",
-                                                    task_configs=[first_task_cfg,
-                                                                  second_task_cfg])
+                                         task_configs=[first_task_cfg,
+                                                       second_task_cfg])
 
 Config.export("config_07.toml")
```
Comment (jrobinAV) [inline docs/knowledge_base/tutorials/scenario_management_overview/step_07/job_execution.py]:
```suggestion
Config.export("config.toml")
```

File: docs/knowledge_base/tutorials/scenario_management_overview/step_07/index.md
Comment (jrobinAV) [inline docs/knowledge_base/tutorials/scenario_management_overview/step_07/index.md:1]: title `entire code` should be H1, no?

File: docs/knowledge_base/tutorials/scenario_management_overview/step_07/job_execution_toml.py
Comment (jrobinAV) [inline docs/knowledge_base/tutorials/scenario_management_overview/step_07/job_execution_toml.py:1]: PLease rename config_07.toml into config.toml within the py file.

**Pull request merged**